Apps/W1/File - SharePoint Connector/app/src/SharePointAccount.Table.al

```diff
@@ -0,0 +1,84 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+/// <summary>
+/// Holds the information for all file accounts that are registered via the SharePoint connector
+/// </summary>
+table 80300 "SharePoint Account"
+{
+    Access = Internal;
+
+    Caption = 'SharePoint Account';
+
+    fields
```

Comment:
@IceOnly could you maybe add the 2nd Authentication Type for SharePoint that was added recently .
This would require a few new Fields: Enum SharePointAuthType , a "Certificate Password Key" and a Certificate Password (Blob)

See: https://github.com/microsoft/BCApps/pull/705